dss-api.yml

```diff
@@ -56,13 +56,20 @@ basePath: /v1
 produces:
   - application/json
 securityDefinitions:
-  googAuth:
+  fusilladeAuth:
+    description: >
+      The following claims must be present in the JWT:
+      - 'https://auth.data.humancellatlas.org/group': the user group this user belongs must be hca or public
+      - 'iss': must be https://humancellatlas.auth0.com/ or a trusted google project
+      - 'aud': audience must be https://dev.data.humancellatlas.org/ or https://data.humancellatlas.org/
```

Comment:
This needs to be just one audience, and it should be parameterized. I think to parameterize this, you'll need to refactor https://github.com/HumanCellAtlas/data-store/blob/master/environment to say:
```
DCP_DOMAIN=dev.data.humancellatlas.org
API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
```
and corresponding overrides in staging/integration/prod environment files/config stores.

The audiences are different for each of dev, staging, integration, and prod.

As a general rule, production docs (which this is part of) should never reference any information about non-production environments. In other words "dev.data.humancellatlas.org" is not a string that may appear in prod docs.